src/parse/asp/grammar.go

```diff
@@ -267,6 +267,10 @@ const (
 	Add Operator = '+'
 	// Subtract implements binary - (only works on integers)
 	Subtract = '-'
+	// Multiply implements multiplication between two types
+	Multiply = '×'
```

Comment:
wait a sec, shouldn't this be `'*'`?

Reply:
Nope these aren't used that way - they are just arbitrary constants, and for mild performance reasons it's a rune rather than a string. The map below provides a mapping of the written form to the enum, where the written form could be multiple characters (e.g. `>=`); if you look at the rest of the constants they use single-character versions (like `≥`).